gc/ogc/gx.h

```diff
@@ -362,9 +362,9 @@
 /*! \addtogroup texgentyp Texture coordinate generation type
  * @{
  */
-#define GX_TG_MTX3x4		0			/*!< 2x4 matrix multiply on the input attribute and generate S,T texture coordinates. */
```

Comment:
Wouldn't it make more sense to swap 2x4 and 3x4 here instead of swapping them elsewhere and fixing the comment here to match?

Reply:
I'm confused.

Reply:
You can turn this into a two line diff instead of a four line diff by simply changing 0 to be GX_TG_MTX2x4 and 1 to be GX_TG_MTX3x4 instead of changing 0 to be GX_MTX3x4, 1 to be GX_MTX2x4, and changing the comments here.

Reply:
The two enums are unrelated. GXMatrixType having the wrong values is the real fix here, and while I was at it I noticed that the documentation for the GXTexGenType values were wrong as well and swapped those.